workflows/sars-cov-2-variant-calling/sars-cov-2-pe-illumina-wgs-variant-calling/pe-wgs-variation.ga

```diff
@@ -428,7 +428,7 @@
         },
         "8": {
             "annotation": "",
-            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.8+galaxy1",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.9+galaxy1",
```

Comment:
I think that would require also changing:
- `tool_id`
- `tool_shed_repository` -> `changeset_revision`
- `tool_version`
?

Reply:
That's right and that is why it may be safer to change the workflow using the visual editor